dotnetv3/S3/README.md

```diff
@@ -0,0 +1,92 @@
+# Amazon Simple Storage Service (Amazon S3) code examples in C#
+
+## Purpose
+
+Customers of all sizes and industries can use Amazon S3 to store and protect any amount of data for a range of use cases, such as data lakes, websites, mobile applications, backup and restore, archive, enterprise applications, IoT devices, and big data analytics. Amazon S3 provides management features so that you can optimize, organize, and configure access to your data to meet your specific business, organizational, and compliance requirements.
+
+The code examples in this directory demonstrate how to work with Amazon S3 features using the AWS SDK for .NET v3.5 or later.
+
+## Code examples
+
+- [AbortMPUExample](AbortMPUExample/)
```

Comment:
For each code examples, can you include the command in parentheses after the enter? (i.e. - [AbortMPUExample](AbortMPUExample/) (AbortMPUAsync)